cmd/minikube/cmd/config/config.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors All rights reserved.
+Copyright 2021 The Kubernetes Authors All rights reserved.
```

Comment:
This one should stay as 2016, it's `cmd/minikube/cmd/config/list.go` that should updated to 2021 since that's a brand new file.